algorithms/monte_carlo.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional, Dict
+from base.action import Action
+from base.bot import Bot, G
+from base.state import State
+import random
+
+
+class MonteCarlo(Bot):
+    def __init__(self, game: G, n_rollouts: int):
+        self.n_rollouts = n_rollouts
```

Comment:
To jest ok, ale tak naprawdę powinniśmy przenieść się na #8, żeby uspójnić ograniczenia dla graczy.